<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkRigidRegistrationViewControls</class>
 <widget class="QWidget" name="QmitkRigidRegistrationViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>827</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>427</width>
    <height>827</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RigidRegistrationControls</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="TextLabelFixed">
          <property name="text">
           <string>Fixed:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QmitkDataStorageComboBox" name="m_FixedSelector"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="TextLabelMoving">
          <property name="text">
           <string>Moving:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QmitkDataStorageComboBox" name="m_MovingSelector"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QmitkSliderNavigatorWidget" name="timeSlider" native="true"/>
      </item>
      <item>
       <widget class="QCheckBox" name="m_ShowRedGreenValues">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Show Images Red/Green</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_3">
        <item>
         <widget class="QLabel" name="m_OpacityLabel">
          <property name="text">
           <string>Moving Image Opacity:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="m_OpacitySlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="m_SelectedModel">
        <property name="title">
         <string>Select Shown Image</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QRadioButton" name="m_ShowFixedImage">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Fixed Image</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="m_ShowMovingImage">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Moving Image</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="m_ShowBothImages">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Both Images</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_2">
        <item>
         <widget class="QPushButton" name="m_FixedReinit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Reinit Fixed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_MovingReinit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Reinit Moving</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_GlobalReinit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Global Reinit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="m_UndoTransformation">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Undo Transformation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_RedoTransformation">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Redo Transformation</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="m_SaveModel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Save Transformed Moving Image</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_17">
        <item>
         <widget class="QPushButton" name="m_LoadRigidRegistrationParameter">
          <property name="text">
           <string>Load Rigid Registration Parameter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_SaveRigidRegistrationParameter">
          <property name="text">
           <string>Save Rigid Registration Parameter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="m_RigidTransform">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Automatic</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QScrollArea" name="frame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>365</width>
               <height>328</height>
              </rect>
             </property>
             <widget class="QmitkRigidRegistrationSelectorView" name="qmitkRigidRegistrationSelector1" native="true">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>361</width>
                <height>321</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="TabPage">
         <attribute name="title">
          <string>Manual</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QFrame" name="m_ManualFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QGroupBox" name="interactiveTranslationBox">
               <property name="title">
                <string>Interactive Translations</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <layout class="QHBoxLayout" name="_8">
                  <item>
                   <layout class="QVBoxLayout" name="_9">
                    <item>
                     <widget class="QLabel" name="xTranslationLabel">
                      <property name="font">
                       <font>
                        <weight>50</weight>
                        <bold>false</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>x-Direction (Frontal):</string>
                      </property>
                      <property name="wordWrap">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="yTranslationLabel">
                      <property name="font">
                       <font>
                        <weight>50</weight>
                        <bold>false</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>y-Direction (Sagittal):</string>
                      </property>
                      <property name="wordWrap">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="zTranslationLabel">
                      <property name="font">
                       <font>
                        <weight>50</weight>
                        <bold>false</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>z-Direction (Transversal):</string>
                      </property>
                      <property name="wordWrap">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="_10">
                    <item>
                     <widget class="QSlider" name="m_XTransSlider">
                      <property name="cursor">
                       <cursorShape>ArrowCursor</cursorShape>
                      </property>
                      <property name="toolTip">
                       <string/>
                      </property>
                      <property name="minimum">
                       <number>-256</number>
                      </property>
                      <property name="maximum">
                       <number>256</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSlider" name="m_YTransSlider">
                      <property name="minimum">
                       <number>-256</number>
                      </property>
                      <property name="maximum">
                       <number>256</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSlider" name="m_ZTransSlider">
                      <property name="minimum">
                       <number>-256</number>
                      </property>
                      <property name="maximum">
                       <number>256</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="automaticTranslationBox">
               <property name="title">
                <string>Automatic Allign Image Centers</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QPushButton" name="m_AutomaticTranslation">
                  <property name="font">
                   <font>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Automatic Align Image Centers</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="interactiveRotationBox">
               <property name="title">
                <string>Interactive Rotations</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QHBoxLayout" name="_13">
                  <item>
                   <layout class="QVBoxLayout" name="_14">
                    <item>
                     <widget class="QLabel" name="xRotationLabel">
                      <property name="font">
                       <font>
                        <weight>50</weight>
                        <bold>false</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>x-Axis (Frontal):</string>
                      </property>
                      <property name="wordWrap">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="yRotationLabel">
                      <property name="font">
                       <font>
                        <weight>50</weight>
                        <bold>false</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>y-Axis (Sagittal):</string>
                      </property>
                      <property name="wordWrap">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="zRotationLabel">
                      <property name="font">
                       <font>
                        <weight>50</weight>
                        <bold>false</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>z-Axis (Transversal):</string>
                      </property>
                      <property name="wordWrap">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="_15">
                    <item>
                     <widget class="QSlider" name="m_XRotSlider">
                      <property name="cursor">
                       <cursorShape>ArrowCursor</cursorShape>
                      </property>
                      <property name="toolTip">
                       <string/>
                      </property>
                      <property name="minimum">
                       <number>-20</number>
                      </property>
                      <property name="maximum">
                       <number>20</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSlider" name="m_YRotSlider">
                      <property name="minimum">
                       <number>-20</number>
                      </property>
                      <property name="maximum">
                       <number>20</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSlider" name="m_ZRotSlider">
                      <property name="minimum">
                       <number>-20</number>
                      </property>
                      <property name="maximum">
                       <number>20</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_CalculateTransformation">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Calculate Transformation</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_4">
        <item>
         <widget class="QLabel" name="m_OptimizerValueLabel">
          <property name="text">
           <string>Optimizer Value:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="m_OptimizerValueLCD">
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="numDigits">
           <number>10</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="m_StopOptimization">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop Optimization</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkRigidRegistrationSelectorView</class>
   <extends>QWidget</extends>
   <header location="global">src/internal/QmitkRigidRegistrationSelectorView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkSliderNavigatorWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkSliderNavigatorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">mitkDataTreeNode.h</include>
  <include location="local">mitkDataTree.h</include>
  <include location="local">mitkBaseData.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
